### PR TITLE
Final review of package

### PR DESCRIPTION
**Pull request by howardbaik**
Here are all the changes made:

- `create_course()` and `get_course_list()`: Printed the `owner_id` argument when user does not explicitly supply it.
- Added @return roxygen tags to functions that explicitly return a list or data frame (I may have missed some functions, may need to doublecheck)
- If possible, return a dataframe instead of a list with one data frame inside. (should we use tibbles instead for a better viewing experience? @cansavvy)
- In `create_course()` and `create_material()`,  add `load_url` argument and `if (load_url) {browseURL(result_list$alternateLink)}` to give user option to load URL into browser
- Addressed R CMD checks by adding files to .Rbuildignore.

### CODE REVIEW

File: R/courses.R [hunk 1]
```diff
@@ -3,17 +3,20 @@
...
-#'
 #' owner_id <- get_owner_id()
 #' course_df <- get_course_list(owner_id)
 #' }
 get_course_list <- function(owner_id = get_owner_id()$id) {
   if (is.null(owner_id)) {
     stop("Need to provide owner_id from which to retrieve a list of courses from")
   }
+  if (missing(owner_id)) {
+    message("Using `owner_id = \"", get_owner_id()$id, "\"`")
```
Comment (cansavvy) [inline R/courses.R:18]: Love it. Thanks!

File: R/courses.R [hunk 2]
```diff
@@ -33,24 +36,36 @@ get_course_list <- function(owner_id = get_owner_id()$id) {
...
 #' @importFrom jsonlite fromJSON
 #' @importFrom utils browseURL
+#' @return List of metadata of newly created course.
 #' @export
 #'
 #' @examples \dontrun{
 #'
 #' owner_id <- get_owner_id()
-#' course_df <- create_course(owner_id, name = "New course")
+#' course_df <- create_course(name = "Computer Science 101")
```
Comment (cansavvy) [inline R/courses.R:58]: I'm not sure as a default name this makes sense? What was your rationale here?

File: R/courses.R [hunk 3]
```diff
@@ -33,24 +36,36 @@ get_course_list <- function(owner_id = get_owner_id()$id) {
...
 #' }
-create_course <- function(owner_id = get_owner_id()$id, name = NULL) {
+create_course <- function(owner_id = get_owner_id()$id, name = "New course", load_url = TRUE) {
+  if (missing(owner_id)) {
+    message("Using `owner_id = \"", get_owner_id()$id, "\"`")
+  }
+  if (missing(name)) {
+    message("Using `name = \"", name, "\"`")
```
Comment (cansavvy) [inline R/courses.R:65]: Yes yes. Perfect! Love it.

File: R/courses.R [hunk 4]
```diff
@@ -174,5 +188,5 @@ delete_course <- function(course_id) {
   result_content <- content(result, "text")
   result_list <- fromJSON(result_content)
 
-  return(result_list)
+  message("Course successfully deleted.")
```
Comment (cansavvy) [inline R/courses.R:191]: :+1:

File: README.Rmd [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+library(rgoogleclassroom)
+
+authorize()
+```
+
+### Basics
+
+There are different objects on the Google API:
+  
+  - courses
```
Comment (cansavvy) [inline README.Rmd]:
```suggestion
- [courses](https://developers.google.com/classroom/reference/rest/v1/courses)
```

Comment (cansavvy) [reply README.Rmd]: Oh we maybe should link to the Google API docs here.

**Review by cansavvy — APPROVED** [review]
I only have minor comments and questions. Thanks so much for doing this work @howardbaek This package is looking better already!

File: R/courses.R (hunk 2 above)
Comment (howardbaik) [reply R/courses.R:58]: Oh this is just a name used in the example code. The default name is set to "New course"

**Comment by howardbaik** [comment]
@cansavvy  I just added links to Google API docs in commit https://github.com/datatrail-jhu/rgoogleclassroom/pull/7/commits/f239254bb4bf9835760b548510ecac3baf56888c. Do you mind reviewing these links?

**Comment by cansavvy** [comment]
@howardbaek Perfect! Thanks for doing this!

**Pull request merged**